packages/web-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -131,6 +132,11 @@ export const optionStyles: (
       margin-inline-end: 1ch;
     }
   `.withBehaviors(
+    new DirectionalStyleSheetBehavior(null, css`
+      :host::before {
+        right: calc((${focusStrokeWidth} - ${strokeWidth}) * 1px);
```

Comment:
Can we use `inset-inline` instead?

Reply:
I think we can look to leverage that in the coming v3 implementation, but given we don't use it anywhere I'm aware in the stable version I'd like to solve without for now.